pkgs/by-name/bl/blenderfarm/package.nix

```diff
@@ -0,0 +1,127 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+, xz
+, pcre
+, libX11
+, libICE
+, libSM
+, autoPatchelfHook
+, bintools
+, fixDarwinDylibNames
+, darwin
+, fontconfig
+, libgdiplus
+, libXrandr
+, glib
+, writeShellScriptBin
+, blender
+}:
+let
+
+  # blenderfarm (client) will run from the current workdir.
+  # It needs to create files there, so it cannot be in the nix store.
+  # We also need to create some files there so it can work with its
+  # server part.
+  USERHOMEDIR = "~/blenderfarm";
```

Comment:
Why can't we use `$(mktemp -d)` instead?

Reply:
Because we need this directory to persist over restarts of blenderfarm. It saves the configuration, blenderfiles and renders there

Reply:
Hm. Might be better to use something like `.local/share/blenderfarm` or `.local/state/blenderfarm` for that purpose then